src/celeritas/ext/GeantImporter.cc

```diff
@@ -222,32 +222,46 @@ fill_vec_import_scint_comp(detail::GeantMaterialPropertyGetter& get_property,
         ImportScintComponent comp;
         get(&comp.yield_frac, "YIELD", ImportUnits::inv_mev);
 
-        // Custom-defined properties not available in G4MaterialPropertyIndex
-        for (auto&& [prop, label] : {
-                 std::pair{&comp.lambda_mean, "LAMBDAMEAN"},
-                 std::pair{&comp.lambda_sigma, "LAMBDASIGMA"},
-             })
-        {
-            if (get_property(
-                    prop, "CELER_" + prefix + label, comp_idx, ImportUnits::len))
-            {
-                any_found = true;
-            }
-            else if (get(prop, label, ImportUnits::len))
-            {
-                CELER_LOG(warning)
-                    << "Deprecated property name " << prefix << label
-                    << ": use CELER_" << prefix << label;
-            }
-        }
-
         // Rise time is not defined for particle type in Geant4
         get(&comp.rise_time, "RISETIME", ImportUnits::time);
         get(&comp.fall_time, "TIMECONSTANT", ImportUnits::time);
 
-        if (any_found)
+        using ScintOption
+            = std::variant<ImportGaussianScintComponent, inp::Grid>;
+        ScintOption scint_data{ImportGaussianScintComponent{}};
```

Comment:
I suggest this change:
```suggestion
        // Spectrum for this component may be missing (default), gaussian (Celeritas-only), or grid
        using ScintSpectrumComponent
            = std::variant<std::monostate, ImportGaussianScintComponent, inp::Grid>;
        ScintSpectrumComponent component;
```
The default variant ("monostate", sort of like a built-in `std::optional`) eliminates the need for `any_found`, so you should delete that afterward.